hardhat.config.ts

```diff
@@ -5,6 +5,27 @@ import "@nomicfoundation/hardhat-ethers";
 import "@nomicfoundation/hardhat-chai-matchers";
 
 const config: HardhatUserConfig = {
+  networks: {
+    // Used for the origin chain started by `yarn chain:origin`
+    origin: {
+      url: "http://127.0.0.1:8545",
+      chainId: 31337,
+    },
+    // Used for the destination chain started by `yarn chain:destination`
+    destination: {
+      url: "http://127.0.0.1:8546",
+      chainId: 31338,
+    },
+    // Used for testing
```

Comment:
Maybe:

```suggestion
    // Used for the origin chain started by `yarn chain:a`
    chainA: {
      url: "http://127.0.0.1:8545",
      chainId: 31337,
    },
    // Used for the destination chain started by `yarn chain:b`
    chainB: {
      url: "http://127.0.0.1:8546",
      chainId: 31338,
    },
    // Used for testing
```

Running this locally it seems that the relationship is symmetric - payments go both ways.

We can extend this config file to include, eg, the endpoints of target EVM testnet environments.

Reply:
This is not important - existing setup is fine and works. Should continue moving forward.

Reply:
That's a fair point! It would be nice if we could use more distinct names than `a` and `b` though.. how about naming them after `alice` and `bob` since in the context of the demo alice and bob always connect to different chains.